vividus-plugin-mobile-app/src/main/resources/properties/defaults/run.properties

```diff
@@ -1,5 +1,3 @@
-spring.profiles.active=mobile_app
```

Comment:
what is the reason for these changes?

Reply:
mobile_app profile is used by default when we add plugin-mobile-app as a dependency, so it should not land in defaults folder